ports/geckolib/gecko_bindings/tools/regen.py

```diff
@@ -0,0 +1,366 @@
+#!/usr/bin/env python
```

Comment:
It seems unfortunate this file doesn't get tidy checked.

https://github.com/servo/servo/blob/1e2b9773d693b19d0c804d16098d15d0945cdd79/python/tidy/servo_tidy/tidy.py#L74